subgraphs/openzeppelin-governor/schema.graphql

```diff
@@ -1,3 +1,39 @@
+type DelegateChange @entity {
+  "Unique entity used to keep track of voting power delta"
+  id: ID!
+  "Governance frameworkId"
+  governanceFrameworkId: String
```

Comment:
Could we link it to the token address instead? It'll be closer to the existing GAB tables and more flexible for dual token DAOs like Aave